config/locales/en/mailers.yml

```diff
@@ -58,8 +58,7 @@ en:
       add_to_address_book_title: Add this email to your address book
       add_to_address_book_body: >-
```

Comment:
I was momentarily confused by the naming of these two owing to the existence of `services/address_book.rb`.